test/e2e/e2e.bats

```diff
@@ -6,3 +6,21 @@ source test/e2e/helpers.sh
 	result="$(shp --help)"
 	[ ! -z "$result" ]
 }
+
+@test "shp build create succesfully run with 1 argument" {
+	build=$(randomDnsSubdomainName)
```

Comment:
In terms of naming, I think calling it `name` would be more descriptive. Or maybe `build_name`.